configs/setups/awiesm3/awiesm3.yaml

```diff
@@ -0,0 +1,782 @@
+                "soil_layer_type_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/slt/slt_${mygrid}.nc
+                "tmp_rc_file_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/lpjg_steps.rc.j2
+                "CO2_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/cmip6_forcing/mole_fraction_of_carbon_dioxide_in_air_input4MIPs_lpjg.nc
+                "guess_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/guess.ins.j2
+                "ecearth_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/ecearth.ins
+                "globsoil_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/global_soiln.ins
+                "lpjg_cmip6_output_ins_[[proc_list-->PROC_NUM]]": ${namelist_dir}/lpjg_default_cmip6_output.ins
+                #"ece_gridlist_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/gridlists/gridlist_TL159_CORE2.txt #is this the correct file for ece_gridlist.txt?
+                "ece_gridlist_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/gridlists/gridlist_TL255_CORE2.txt #is this the correct file for ece_gridlist.txt?
+                temp_namcouple: /home/a/a270270/temp_confiles/namcouple 
+                "mock_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/dirmaker
+
+        forcing_in_work:
+                soil_layer_type: slt_${mygrid}.nc
+                tmp_rc_file: lpjg_steps.rc
+                CO2: mole_fraction_of_carbon_dioxide_in_air_input4MIPs_lpjg.nc
+                temp_namcouple: namcouple
+                dir_landuse: /landuse/dirmaker
+                dir_fire: /fire/dirmakei
+                ins_files: /*.ins
+                ece_gridlist: ece_gridlist.txt #is this the correct file?
+                lpjg_cmip6_output_ins: lpjg_cmip6_output.ins
+                "all_ins_[[proc_list-->PROC_NUM]]": runPROC_NUM/*.ins 
+                "soil_layer_type_[[proc_list-->PROC_NUM]]": runPROC_NUM/slt_${mygrid}.nc
+                "tmp_rc_file_[[proc_list-->PROC_NUM]]": runPROC_NUM/lpjg_steps.rc
+                "CO2_[[proc_list-->PROC_NUM]]": runPROC_NUM/mole_fraction_of_carbon_dioxide_in_air_input4MIPs_lpjg.nc
+                "guess_ins_[[proc_list-->PROC_NUM]]": runPROC_NUM/guess.ins
+                "ecearth_ins_[[proc_list-->PROC_NUM]]": runPROC_NUM/ecearth.ins 
+                "globsoil_ins_[[proc_list-->PROC_NUM]]": runPROC_NUM/global_soiln.ins  
+                "lpjg_cmip6_output_ins_[[proc_list-->PROC_NUM]]": runPROC_NUM/lpjg_cmip6_output.ins
+                "ece_gridlist_[[proc_list-->PROC_NUM]]": runPROC_NUM/ece_gridlist.txt #is this the correct file?  
+                "mock_[[proc_list-->PROC_NUM]]": runPROC_NUM/output/stdo_stde.log
+                 
+        choose_general.run_number:
+                1:
+                        input_sources:
+                                vegin: ${lpj_guess.input_dir}/vegin/vegin_${mygrid}.nc
+                                "VegInit_[[proc_list-->PROC_NUM]]": ${lpj_guess.input_dir}/vegin/vegin_${mygrid}.nc
```

Comment:
This is correctm even if the rhs does not have PROC_NUM because the key needs to be expanded so that this files is written in work (`input_in_work`) `PROC_NUM` number of times.

Reply:
Yes,you are right